Projects/Foundations/Schemas/Docs-Topics/B/Buddha/Buddha-Data-Mine/buddha-data-mine-001-candle-patterns-buddah-indicator.json

```diff
@@ -52,49 +122,126 @@
+            "text": "The corresponding layer panel features the following properties for each candle:",
+            "translations": [
+                {
+                    "language": "DE",
+                    "text": "Das entsprechende Ebenen Panel weist für jede Kerze die folgenden Eigenschaften auf:",
+                    "updated": 1642002332543
+                }
+            ]
         },
         {
             "style": "Table",
             "text": "| Indicator Properties | Description |\n| Name | The name of the basic candle. |\n| Size | The relative size of the candle: Short or Long. |\n| Upper Shadow % | The percentage that the upper shadow covers in the total length of the candle. |\n| Body % | The percentage that the body represents in the total length of the candle. |\n| Lower Shadow % | The percentage that the lower shadow represents in the total length of the candle. |\n| Nbodies\tThe number of previous candles used to calculate the body average. |\n| dojiFactor % | A limit to the size of the body expressed as a percentage of the total length for a candle to be considered a doji. |\n| sizeFactor % | A factor used to calculate the orange line, which determines the relative size of candles, expressed as a percentage of the average length of the last few candles. |\n| Nlengths | The number of previous candles used to calculate the average candle lengths. |\n| Average Size Method | The method used to calculate the average of the total lengths of previous candles. |",
-            "updated": 1611660199319
+            "updated": 1611660199319,
+            "translations": [
+                {
+                    "language": "DE",
+                    "text": "| Indikatoreigenschaften | Beschreibung|\n| Name | Bezeichnung der Basiskerze |\n| Size | Die relative Größe der Kerze: Kurz oder lang. |\n| Upper Shadow % | Der Prozentsatz, den der obere Schatten an der Gesamtlänge der Kerze ausmacht. |\n| Body % | Der Prozentsatz, den der Körper an der Gesamtlänge der Kerze ausmacht. |\n| Lower Shadow % | Der Prozentsatz, den der untere Schatten an der Gesamtlänge der Kerze ausmacht. |\n| Nbodies\t| Die Anzahl der vorherigen Kerzen, die zur Berechnung des Körper Durchschnitts verwendet werden. |\n| dojiFactor % | Ein Grenzwert für die Größe des Körpers, ausgedrückt als Prozentsatz der Gesamtlänge einer Kerze, die als Doji gilt. |\n| sizeFactor % | Ein Faktor, der zur Berechnung der orangefarbenen Linie verwendet wird, die die relative Größe der Kerzen bestimmt, ausgedrückt als Prozentsatz der durchschnittlichen Länge der letzten paar Kerzen. |\n| Nlengths | Die Anzahl der vorherigen Kerzen, die zur Berechnung der durchschnittlichen Kerzenlängen verwendet werden. |\n| Average Size Method | Die Methode, die zur Berechnung des Durchschnitts der Gesamtlängen der vorherigen Kerzen verwendet wird. |",
+                    "updated": 1642436998675
```

Comment:
@harrellbm do tables always look like that in code?

Reply:
Yes those are his tables always look. They are in markdown style format.  I haven't looked into how the code is read specifically but if you ever write a table in markdown it will look exactly the same